src/vm/amd64/crthelpers.S

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Geoff Norton. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+.intel_syntax noprefix
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+// JIT_MemSet/JIT_MemCpy
+//
+// It is IMPORANT that the exception handling code is able to find these guys
+// on the stack, but on non-windows platforms we can just defer to the platform
+// implementation.
+// 
+
+LEAF_ENTRY JIT_MemSet, _TEXT
+        test rdx, rdx
+        jz   Exit_MemSet
+
+        cmp  byte ptr [rdi], 0
+
+        jmp  C_FUNC(memset)
+
+Exit_MemSet:
+        ret
+
+LEAF_END JIT_MemSet, _TEXT
```

Comment:
We should use LEAF_END_MARKED here and remove JIT_MemSet_End from the UnixStubs.cpp too. It is what actually allows the exception handling handling code to able to find these guys.